components/zcash_protocol/src/consensus.rs

```diff
@@ -657,15 +648,16 @@ impl BranchId {
             BranchId::Canopy => params
                 .activation_height(NetworkUpgrade::Canopy)
                 .map(|lower| (lower, params.activation_height(NetworkUpgrade::Nu5))),
-            BranchId::Nu5 => params.activation_height(NetworkUpgrade::Nu5).map(|lower| {
+            BranchId::Nu5 => params
+                .activation_height(NetworkUpgrade::Nu5)
+                .map(|lower| (lower, params.activation_height(NetworkUpgrade::Nu6))),
+            BranchId::Nu6 => params.activation_height(NetworkUpgrade::Nu6).map(|lower| {
```

Comment:
Codecov complains about this code has not being tested, which is correct but i think we can't add tests for this until we have the Nu6 activation height ?

Reply:
Can be resolved as in this case the failure is informative.

Reply:
> we can't add tests for this until we have the Nu6 activation height ?

That's not true; this code just needs _some_ `P: Parameters`, not necessarily the mainnet or testnet parameters.

That being said, we only use codecov for informational purposes.